app/controllers/api/v1/moves_controller.rb

```diff
@@ -22,10 +22,21 @@ def show
       def create
         move = Move.new(move_attributes)
         authorize!(:create, move)
-        move.save!
-        move.documents.each { |doc| doc.update(move: move) }
-        Notifier.prepare_notifications(topic: move, action_name: 'create')
-        render_move(move, 201)
+        if move.save
+          move.documents.each { |doc| doc.update(move: move) }
```

Comment:
What about moving this to `move`,
with a name like `move.update_documents`:

Advantages:
- you have less noisy in the controller
- it is testable (in case you need a test)